afs/afs-ws/afs-ws-server/src/main/java/com/powsybl/afs/ws/server/AppStorageServer.java

```diff
@@ -595,13 +592,25 @@ public Response getNodeInfo(@ApiParam(value = "File system name") @PathParam("fi
     @Produces(MediaType.APPLICATION_JSON)
     @Path("fileSystems/{fileSystemName}/tasks")
     public Response startTask(@PathParam("fileSystemName") String fileSystemName,
-                              @QueryParam("projectFileId") String projectFileId) {
+                              @QueryParam("projectFileId") String projectFileId,
+                              @QueryParam("projectId") String projectId,
```

Comment:
Is project id really necessary as it from a project file it is possible to reach the project by getting parents ?

Reply:
Yes, but this parameter is to be used only with the next one (name).

I added two parameters to the only existing one (projectFileId), to keep backward compatibility with previous way to create a task.
So now you have to provide either a projectFileId or a projectId + a name.